### PR TITLE
chore(EMI-1969): Collector signals schema

### PR DESCRIPTION
**Pull request by erikdstock**
The type of this PR is: **chore**

This PR resolves relates to [EMI-1969] and creates new properties for interactions with an artwork (page, screen or grid/collection item) that has collector signals visible on it. 

### Description

Track commercial interactions with the following values based on what is visible on the screen/artwork collection item:
```ts
/*
  Possible signal labels today:
  | "Limited-Time Offer"
  | "Showing now"
  | "Bidding live now"
  | "Time left to bid"
*/
interface SomeInteraction  {
  signal_labels?: string[]
  signal_lot_watcher_count?: number
  signal_bid_count?: number
}
```
Added to:
### Click events
- [x] ClickedMainArtworkGrid
- [x] ClickedArtworkGroup
- [x] ClickedAuctionGroup
- [x] ClickedBuyNow
- [x] ClickedBid (Migrated from usage in DeprecatedSchema - will need to be updated in artsy/force)
- [x] ClickedContactGallery (Migrated from usage in DeprecatedSchema - will need to be updated in artsy/force)
### Tap events
- [x] TappedMainArtworkGrid
- [x] TappedArtworkGroup
- [x] TappedAuctionGroup
- [x] TappedBuyNow
- [x] TappedBid
- [x] TappedContactGallery

## Checklist
- [x] If I've added a new file to the tree I've exported it from the common [`index.ts`](https://github.com/artsy/cohesion/blob/main/src/Schema/Events/index.ts)
- [x] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[EMI-1969]: https://artsyproduct.atlassian.net/browse/EMI-1969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Schema/Events/Click.ts [hunk 1]
```diff
@@ -176,10 +177,13 @@ export interface ClickedArtistSeriesGroup extends ClickedEntityGroup {
...
  *    horizontal_slide_position: 1,
  *    type: "thumbnail"
+ *    signal_labels: ["Limited-Time Offer"],
  *  }
  * ```
  */
-export interface ClickedArtworkGroup extends ClickedEntityGroup {
+export interface ClickedArtworkGroup
+  extends ClickedEntityGroup,
+    CollectorSignalsValues {
```
Comment (damassi) [inline src/Schema/Events/Click.ts:186]: No cleverness in this codebase! Don't want to extend or anything like that, especially in regards to mixing types. Keep this as boring as possible 🙏

File: src/Schema/Events/Click.ts [hunk 2]
```diff
@@ -202,7 +206,9 @@ export interface ClickedArtworkGroup extends ClickedEntityGroup {
  *  }
  * ```
  */
-export interface ClickedAuctionGroup extends ClickedEntityGroup {
+export interface ClickedAuctionGroup
+  extends ClickedEntityGroup,
+    CollectorSignalsValues {
```
Comment (damassi) [inline src/Schema/Events/Click.ts:211]: Ditto here and elsewhere. We can surely achieve the same ends without crossing wires.

**Review by damassi — CHANGES_REQUESTED** [review]
Happy to pair on these change requests! We shouldn't be extending the tracking schema in this way.

**Review by damassi — APPROVED** [review]
Thanks for the updates 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`v4.196.0`](https://github.com/artsy/cohesion/releases/tag/v4.196.0) :rocket:

**Pull request merged**